src/OrbitClientGgp/ClientGgp.cpp

```diff
@@ -182,7 +182,12 @@ ErrorMessageOr<void> ClientGgp::LoadModuleAndSymbols() {
   OUTCOME_TRY(module_infos, process_client_->LoadModuleList(target_process_.pid()));
 
   LOG("List of modules");
+  std::string target_process_build_id;
   for (const ModuleInfo& info : module_infos) {
+    // TODO(dimitry): eventually we want to get build_id directly from ProcessData
```

Comment:
Should this be a bug?

Reply:
it is part of the draft CL (start_capture sends build_id for the executable).